clients/src/main/java/org/apache/kafka/common/config/AbstractConfig.java

```
@@ -185,17 +185,14 @@ public void logUnused() {
         List<String> klasses = getList(key);
         List<T> objects = new ArrayList<T>();
         for (String klass : klasses) {
-            Class<?> c;
+            Object o;
             try {
-                c = Class.forName(klass);
+                o = Utils.newInstance(klass, t);
             } catch (ClassNotFoundException e) {
-                throw new ConfigException(key, klass, "Class " + klass + " could not be found.");
+                throw new KafkaException(klass + " ClassNotFoundException exception occured");
```

Why are we swallowing the original exception?